pages/contact/Contact.module.css

```diff
@@ -1,10 +1,13 @@
 .contactPage {
-    margin-top: 50px;
+    margin-top: 60px;
     margin-bottom: 50px;
     display: flex;
     justify-content: flex-start;
     align-items: center;
     flex-direction: column;
+    min-height: calc(100vh - 300px);
+    background: rgb(51,51,51);
+    background: linear-gradient(0deg, rgba(51,51,51,1) 0%, rgba(16,16,16,1) 75%); 
```

Comment:
Let's change this to a more symmetrical gradient. It doesn't display too well on my 13" display. `linear-gradient(0deg, rgba(51,51,51,1) 33%, rgba(16,16,16,1) 66%); ` should look nice.

Reply:
fixed in push: fix gradient